### ISSUE TITLE
Torchscript unsupported inplace operator 

### ISSUE THREAD

**Issue by triwahyuu** [lutzroeder/netron#473]
I can't open a model that consist of PyTorch inplace operator `torch.Tensor.sub_` and `torch.Tensor.div_`. The problem actually could be solved by using non-inplace operator but I need to use the inplace one.

**Comment by lutzroeder** [comment]
Can yo u share the model file?

**Comment by triwahyuu** [comment]
it is just a simple normalization function that I turn into `nn.Module`.
[norm_inplace.pt.zip](https://github.com/lutzroeder/netron/files/4516471/norm_inplace.pt.zip)

```
def normalize(tensor, mean, std):
    tensor.sub_(mean[None, :, None, None]).div_(std[None, :, None, None])
    return tensor

class Normalize(nn.Module):
    def __init__(self, mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]):
        super(Normalize, self).__init__()
        self.register_buffer("mean", torch.as_tensor(mean, dtype=torch.float))
        self.register_buffer("std", torch.as_tensor(std, dtype=torch.float))

    def forward(self, input: torch.Tensor) -> torch.Tensor:
        x = normalize(x, self.mean, self.std)
        return x
```

**Comment by lutzroeder** [comment]
[norm_inplace.pt.zip](https://github.com/lutzroeder/netron/files/4516621/norm_inplace.pt.zip)
